coq-builtin.elpi

```diff
@@ -775,6 +780,11 @@ type coq.inline.no coq.inline. % Coq's [no inline] (aka !)
 type coq.inline.default coq.inline. % The default, can be omitted
 type coq.inline.at int -> coq.inline. % Coq's [inline at <num>]
 
+% [coq.env.fresh-global-id The ID one wants The fresh id closer to ID,
+% starting from 1] Generates an id which is not taken (in the current
+% module)
```

Comment:
I know this is automatically generated, but this is not really readable.

Reply:
oops, dunno what went wrong

Reply:
fixed